src/tablestore.rs

```diff
@@ -389,7 +402,7 @@ mod tests {
     #[tokio::test]
     async fn test_sst_writer_should_write_sst() {
         // given:
-        let os = Arc::new(object_store::memory::InMemory::new());
+        let os: Arc<dyn ObjectStore> = Arc::new(object_store::memory::InMemory::new());
```

Comment:
Can you please add at least one test that validates that writes through `tablestore` caches the sst on local disk? In addition to tests at `db` level, this would be easier to debug.